packages/viewer/src/modules/SceneObjectManager.js

```diff
@@ -13,10 +13,11 @@ import { getConversionFactor } from './converter/Units'
  * Manages objects and provides some convenience methods to focus on the entire scene, or one specific object.
  */
 export default class SceneObjectManager {
+  views = []
```

Comment:
@AlexandruPopovici had to add this as a public class field, otherwise the exported TS type won't have it